Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -9030,6 +9030,96 @@ def inner(client, args):
     return dec
 
 
+def get_pack_version(pack_name=''):
+    """
+    Get a pack version.
+    The version can be retrieved either by a pack name, script name or integration name.
```

Comment:
Sounds like from this I can set `pack_name = some_integration_name`. Is that correct? Does it then make sense that  `query_type = 'pack'` for an integration_name?

Reply:
well, I guess it hasn't been explained very well in the doc-string. 
My intention is that only if you want to query by a pack name - then you should pass pack_name as an input for the function, otherwise you can just call the function without this pack_name as input (will get default to empty string)

Reply:
ok, so lets update the docstring so its more clear, either a pack name, or leave it empty for current pack

Reply:
done, take a look at the current doc-string and let me know if that's understandable.